velox/functions/prestosql/Map.cpp

```diff
@@ -32,6 +32,14 @@ class MapFunction : public exec::VectorFunction {
       const TypePtr& outputType,
       exec::EvalCtx& context,
       VectorPtr& result) const override {
+    // No arguments case (empty map).
```

Comment:
Would you update documentation to add this signature?

Reply:
@mbasmanova 
I'm not sure what documentation you are talking about.
Do we have a page listing ALL signatures of all functions?

Reply:
@spershin Yes, we have documentation that should list all signatures.

https://facebookincubator.github.io/velox/functions/presto/map.html#map

Reply:
> @spershin Yes, we have documentation that should list all signatures.
> 
> https://facebookincubator.github.io/velox/functions/presto/map.html#map

It seems map() is already in the documentation, just not supported.